frontend/src/api/project.js

```diff
@@ -59,6 +59,12 @@ const updateProject = async (projectId, options) => {
         return res.data
     })
 }
+const rollbackProject = async (projectId, snapshotId) => {
+    const data = {
+        snapshotId: snapshotId
```

Comment:
```suggestion
        snapshot: snapshotId
```

For the external API, the property should be snapshot rather than snapshotId. This is more consistent with other places we pass object ids over the api.